sources/Makefile

```diff
@@ -67,13 +67,13 @@ export CRA_RESPONSE_FOLDER := $(or $(CRA_RESPONSE_FOLDER), CRA-Response)
 export CRA_PROGRAM_AREA_CODE := $(or $(CRA_PROGRAM_AREA_CODE), )
 export CRA_ENVIRONMENT_CODE := $(or $(CRA_ENVIRONMENT_CODE), )
 # Versions used by camunda-docker compose-core.yml during the make camunda.
-# Versions upgraded following the combination available on https://github.com/camunda/camunda-platform/blob/main/.env
-export CAMUNDA_CONNECTORS_VERSION := $(or $(CAMUNDA_CONNECTORS_VERSION), 8.6.9)
-export CAMUNDA_PLATFORM_VERSION := $(or $(CAMUNDA_PLATFORM_VERSION), 8.6.12)
-export ELASTIC_VERSION := $(or $(ELASTIC_VERSION), 8.17.1)
+# Versions upgraded following the combination available at https://github.com/camunda/camunda-distributions/releases
+export CAMUNDA_CONNECTORS_VERSION := $(or $(CAMUNDA_CONNECTORS_VERSION), 8.7.8)
+# Operate, Zeebe, and tasklist shares the same version below.
+export CAMUNDA_PLATFORM_VERSION := $(or $(CAMUNDA_PLATFORM_VERSION), 8.7.12)
+export ELASTIC_VERSION := $(or $(ELASTIC_VERSION), 8.17.10)
 # Camunda default values for local development.
```

Comment:
Minor. Please update the log here. 

<img width="1322" height="170" alt="Image" src="https://github.com/user-attachments/assets/c57cd16b-dddd-4f20-aa1f-5129a7b812cd" />

Reply:
Updated.